lightningd/chaintopology.c

```diff
@@ -444,6 +459,20 @@ static void json_feerates(struct command *cmd,
 	if (missing)
 		json_add_string(response, "warning",
 				"Some fee estimates unavailable: bitcoind startup?");
+	else {
+		json_object_start(response, "onchain_estimates");
```

Comment:
change `onchain_estimates` to `onchain_fee_estimates` to make it more descriptive

Reply:
Agreed!